Sming/Arch/Esp32/Tools/install.sh

```diff
@@ -31,6 +31,29 @@ case $DIST in
     darwin)
         ;;
 
+    *)
+	_OK=1
+	_COMMANDS=(dfu-util bison flex gperf)
+	for _COMMAND in "${_COMMANDS[@]}"; do
+	    if ! [ -x $(command -v "${_COMMAND}") ]; then
+		_OK=0
+		echo "Install programm ${_COMMAND}"
+	    fi
+	done
+	_INCLUDES=("/usr/include/ffi.h" "/usr/include/ssl/ssl.h")
+	for _INCLUDE in "${_INCLUDES[@]}"; do
+	    if ! [ -f "${_INCLUDE}" ]; then
+		_OK=0
+		echo "Install development package providing ${_INCLUDE}"
+	    fi
+	done
+	if [ $_OK != 1 ]; then
+	    echo "ABORTING"
+	    exit 1
+	fi
+	PACKAGES=()
+        ;;
+
```

Comment:
Seems reasonable, though is an awful lot of script for an edge case. How does Espressif handle non-standard installs of their IDF?

Reply:
The environment variables `ESP_HOME`, `IDF_PATH`, and `IDF_TOOLS_PATH` should be used for non-standard installs of Expressif IDF (ESP32 and ESP8266).